File: Code/3DMANAGER-APP/3DMANAGER-APP.BLL/Models/Filament/FilamentUpdateRequest.cs
using Microsoft.AspNetCore.Http;

namespace _3DMANAGER_APP.BLL.Models.Filament
{
    public class FilamentUpdateRequest
    {
        public int GroupId { get; set; }
        public int FilamentId { get; set; }
        public string FilamentName { get; set; }
        public string? FilamentColor { get; set; }
        public int? FilamentTemperature { get; set; }
        public int FilamentLenght { get; set; }
        public string? FilamentDescription { get; set; }
        public int FilamentCost { get; set; }
        public IFormFile? ImageFile { get; set; }
    }

}

File: Code/3DMANAGER-APP/3dmanager-app.client/src/models/filament/FilamentUpdateRequest.ts
export interface FilamentUpdateRequest {
    groupId: number;
    filamentId: number;
    filamentName: string;
    filamentColor: string;
    filamentTemperature: number;
    filamentLenght: number;
    filamentDescription: string;
    filamentCost: number;
    //imageFile?: File | null;
}